Content: Add very weak pirate ship, Scrapper. (#8015)
endless-sky/endless-sky: data/human/ships.txt

ship "Scrapper"
	sprite "ship/scrapper"
	thumbnail "thumbnail/scrapper"
	attributes
		category "Interceptor"
		"cost" 60000
		"shields" 450
		"hull" 520
		"required crew" 1
		"bunks" 4
		"mass" 62
		"drag" 1.4
		"heat dissipation" .8
		"fuel capacity" 300
		"cargo space" 21
		"outfit space" 125
		"weapon capacity" 12
		"engine capacity" 58
		weapon
			"blast radius" 10
			"shield damage" 80
			"hull damage" 50
			"hit force" 140
	outfits
		"Twin Blaster"
		
		"KP-6 Photovoltaic Panel" 4
		"LP036a Battery Pack"
		
		"X2700 Ion Thruster"
		"X2200 Ion Steering"
		"Hyperdrive"
		
	engine -6 30
	engine 6 30
	gun 0 -31
	leak "leak" 60 50
	explode "tiny explosion" 10
	explode "small explosion" 5

> 	description "The Scrapper is less a model of ship, and more a general reference to ships cobbled together from the remains of scrapped shuttles. While Republic authorities prefer to dismiss them as at best an urban legend, and at worst an almost-illegal fabrication of unlicensed shipyards, these ships are where many aspiring captains from poorer planets get their start."

Reviewer: ```suggestion
	description "Rather than being a specific model of ship, the Scrappers are instead a family of vessels cobbled together from the remains of destroyed shuttles. On the rare occasion that Republic authorities bother to recognize their existence, Scrappers are treated as almost-illegal, patent-infringing death traps. Despite this, these ships are where many captains (and pirates) from poorer backgrounds get their start."
```

Author: I'm not too sure about `remains of destroyed shuttles`, feels too violent and specific..? I like the rest of the suggestion though.

Author: "decommissioned" sounds very military/official to me. I was thinking about it and maybe "remains of shuttle wrecks", but by definition it's pretty much the same as "destroyed", so maybe just "remains of shuttles" might work? Other words I was thinking of might be "plundered", "stolen", "broken down" which are kinda different but should give the same idea.

Reviewer 2: Hmmm, how about merely "broken shuttles" ? I agree that decommissioned sounds rather formal, like ships that were intentionally declared out of service and dismantled properly; whereas I envision these as more being made from shuttles that wore out and just happened to end up parked in a junk spaceship lot. Nothing formal, they're just ships that just stopped getting fixed at some point, and then eventually people tore pieces out of them to stick together to make a new ship.

Some probably were destroyed, but that also sounds deliberate, whereas "broken" is just, well, it's the state they are in.

ship "Scrapper"
	sprite "ship/scrapper"
	thumbnail "thumbnail/scrapper"
	attributes
		category "Interceptor"
		"cost" 60000
		"shields" 450
		"hull" 520
		"required crew" 1
		"bunks" 4
		"mass" 62

> 		"drag" 1.1

Reviewer 2: ```suggestion
		"drag" 1.4
```

This will still go massively faster than a shuttle (Overriding the nav system safeties, I'm guessing?), but not quite so blindly fast. 

As a comparison thing:
The Sparrow, with 176 mass, with 1 chipmunk plasma thruster, has a top speed of 766.
The Scrapper, with 186 mass, with 1 X2700 ion thruster, has a top speed of 752.

Having a heavier, pieced together junk heap of old ships, powered by a low performance ion engine, ship go almost as fast as a brand new fairly modern purpose-built interceptor with decently effective thrusters is... problematic... to me. 

Granted, the shuttle has more engine space than the sparrow, and I don't have a problem with the player being able to push it up to higher speeds than the sparrow. I just don't think it should be stock at such high speeds.

With a drag of 1.4 it is still a fair bit below the shuttle's 1.7, so it'll still be able to "break the safety margins" on the shuttle, but it's not quite as outrageous compared to brand new interceptors. Drag of 1.4 gives it a stock speed of 591, and with greyhound thrusters it can get up to 947 quite easily. I didn't test atomics, but those should have a max speed well over a thousand; so even with a drag of 1.4 it is still going to be a serious contender for racing against other interceptors, fighters, and small ships.

		"heat dissipation" .8
		"fuel capacity" 300
		"cargo space" 20
		"outfit space" 125
		"weapon capacity" 12
		"engine capacity" 58
		weapon
			"blast radius" 10
			"shield damage" 80
			"hull damage" 50
			"hit force" 140
	outfits
		"Twin Blaster"
		
		"nGVF-AA Fuel Cell"
		"LP036a Battery Pack"

> 		"D14-RN Shield Generator"

Reviewer 3: I wish we'd have more ships without a shield generator tbh as a small one often is useless so either we get shield capacitators at some point or just no shield regen

		
		"X2700 Ion Thruster"
		"X2200 Ion Steering"
		"Hyperdrive"
		
	engine -6 30
	engine 6 30
	gun 0 -31
	leak "leak" 60 50
	explode "tiny explosion" 10
	explode "small explosion" 5

> 	description "Scrapper is less of a ship model but rather a type of old shuttle parts barely held together that some might call a ship."

Reviewer 2: ```suggestion
	description "The Scrapper is less a model of ship, and more a general reference to ships cobbled together from the remains of scrapped shuttles. While Republic authorities prefer to dismiss them as at best an urban legend, and at worst an almost-illegal fabrication of unlicensed shipyards, these ships are where many aspiring captains from poorer planets get their start."
```
